ceph-dashboard-pull-requests/config/definitions/ceph-dashboard-pull-requests.yml

```diff
@@ -81,4 +81,11 @@
           - text:
               credential-id: cd-applitools-api-key
               variable: APPLITOOLS_API_KEY
+      - raw:
+          xml: |
+            <com.applitools.jenkins.ApplitoolsBuildWrapper plugin="applitools-eyes@1.13">
+              <serverURL>https://eyes.applitools.com</serverURL>
```

Comment:
Is that really where we want to send results?  I don't know anything about this tool but I guess I assumed it was some sort of self-hosted Web UI accessible via Jenkins.

Reply:
I don't think it's a self hosted web UI, more of an iframe referencing to applitools. This is how it looks like: ![](https://help.applitools.com/hc/article_attachments/360088523632/jenkins.png)

Reply:
Do we have an account or license with Applitools then?  If so, what type?

Reply:
Yeah, we have an open source license of Applitools. And we have passed an api key for jenkins as well.

Reply:
Okay, I manually pushed this to the `ceph-dashboard-pull-requests` job.  Is it possible to run a job to see it get rendered?  Currently the latest job shows this:

![Screenshot at 2021-08-03 13-48-39](https://user-images.githubusercontent.com/11281208/128062316-88e664ba-ff08-4cd9-ad2d-7788af9ff522.png)

Reply:
Ah that's not a good sign, we hoped that applitools would make the test runner public. Thanks for pointing this out!

Reply:
I think it's just because the API key isn't getting populated in the config.

![Screenshot at 2021-08-03 16-13-45](https://user-images.githubusercontent.com/11281208/128079948-1fffe920-e8ee-4896-aff0-019b46b80872.png)

If I change to "foo" in the Web UI then check the Job config, it is "foo" in plaintext :sob:.

```
    <com.applitools.jenkins.ApplitoolsBuildWrapper plugin="applitools-eyes@1.13">
      <serverURL>https://eyes.applitools.com</serverURL>
      <notifyByCompletion>true</notifyByCompletion>
      <applitoolsApiKey>foo</applitoolsApiKey>
    </com.applitools.jenkins.ApplitoolsBuildWrapper>
```

Reply:
Omg

Reply:
Also just found out that I don't get the access denied page if I'm logged into applitools.  I can access the page perfectly.
![image](https://user-images.githubusercontent.com/54525904/128147308-28c45678-3d13-4141-a7f2-cd21aac17e01.png)

Reply:
@djgalloway Applitools granted us (Ceph project) an [Open Source plan](https://applitools.com/open-source/). They're only storing the screenshots and the diffs between those (BTW actually we are also sending lots of tests info to the [Cypress site](https://dashboard.cypress.io/)).


I'll invite you to both sites, so you can access them.

Reply:
@djgalloway I just added you to the Applitools Eyes, and disabled (manually) the Applitools Build Wrapper in the Dashboard job (as it's making fail all the Dashboard tests).

Reply:
> Also just found out that I don't get the access denied page if I'm logged into applitools. I can access the page perfectly.
> ![image](https://user-images.githubusercontent.com/54525904/128147308-28c45678-3d13-4141-a7f2-cd21aac17e01.png)

Oh!  This is good news.

Thanks for the access @epuertat.